api/src/services/gcnotify-service.ts

```diff
@@ -1,33 +1,93 @@
-import axios from 'axios';
+import axios, { AxiosRequestConfig } from 'axios';
 import { ACCESS_REQUEST_ADMIN_EMAIL, ACCESS_REQUEST_APPROVAL_ADMIN_EMAIL } from '../constants/notifications';
 import { IDBConnection } from '../database/db';
 import { ApiError, ApiErrorType, ApiGeneralError } from '../errors/api-error';
 import { IgcNotifyGenericMessage, IgcNotifyPostReturn } from '../interfaces/gcnotify.interface';
 import { AdministrativeRepository } from '../repositories/administrative-repository';
 import { DBService } from './db-service';
 import { KeycloakService } from './keycloak-service';
+import { formatPhoneNumber, makeLoginUrl } from '../utils/string-utils';
+import { ArtifactService } from './artifact-service';
+import { getLogger } from '../utils/logger';
 
-const EMAIL_TEMPLATE = process.env.GCNOTIFY_ONBOARDING_REQUEST_EMAIL_TEMPLATE || '';
-const SMS_TEMPLATE = process.env.GCNOTIFY_ONBOARDING_REQUEST_SMS_TEMPLATE || '';
-const EMAIL_URL = process.env.GCNOTIFY_EMAIL_URL || '';
-const SMS_URL = process.env.GCNOTIFY_SMS_URL || '';
-const API_KEY = process.env.GCNOTIFY_SECRET_API_KEY || '';
+const GC_NOTIFY_REQUEST_ACCESS_SECURE_DOCUMENTS = '4bb42a76-f79b-424f-ad0f-ad3671389ec2'; // @TODO
 
-const config = {
-  headers: {
-    Authorization: API_KEY,
-    'Content-Type': 'application/json'
-  }
-};
+export interface ISubmitArtifactRequestAccess {
+  fullName: string;
+  emailAddress: string;
+  phoneNumber: string;
+  reasonDescription: string;
+  hasSignedAgreement: boolean;
+  artifactIds: number[];
+  pathToParent: string;
+  companyInformation?: {
+    companyName: string;
+    jobTitle: string;
+    streetAddress: string;
+    city: string;
+    postalCode: string;
+  };
+  professionalOrganization?: {
+    organizationName: string;
+    memberNumber: string;
+  };
+}
+
+interface IGcNotifyArtifactRequestAccess {
```

Comment:
nit: Should this live in the gcnotify service? Looks like its only referenced there

Reply:
somehow I read this as not the gcnotfiy service... disregard..